### PR TITLE
Add segwit upgrade guide

### PR DESCRIPTION
**Pull request by harding**
This adds a guide to upgrading (or not upgrading) to segwit for miners, node operators, wallet users, and Bitcoin devs.  

- A link to this guide is added to the FAQ menu.
- A link is *not* added to the main page, as it seems like we'll be adding a link there soon enough for the 0.13.1 release.
- All Markdown reference-style links to this document are defined, as verified by searching the rendered document for the string `][` that is indicative of an undefined link.
- It is expected that the release blog for 0.13.1 will link to this document; however, I think it is safe to merge this document prior to the release if everyone is ok with it's content (which is why it's in a separate PR from the upcoming PR for the release blog).

A preview screenshot may be found here: http://imgur.com/a/DZGAg

Several people provided suggestions on this in draft form, for which I am very grateful.  Any errors that appear in this version are entirely my fault.

### CODE REVIEW

File: _posts/en/pages/2016-10-26-segwit-upgrade-guide.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
+---
+version: 1
```
Comment (btcdrak) [inline _posts/en/pages/2016-10-26-segwit-upgrade-guide.md]: double `version` (again at the bottom)

**Comment by kanzure** [comment]
ACK 290d0c43739d9b7e394eb76ac3d05da019939ad4. Also possibly consider adding a link to `/en/2016/01/26/segwit-benefits/`. Looking good.

**Comment by btcdrak** [comment]
Converted to a blog post, makes it more accessible. Agree with @kanzure to add link to segwit benefits.

**Comment by btcdrak** [comment]
@harding Thank you very much

**Comment by laanwj** [comment]
post-merge ACK, thanks!

**Comment by kanzure** [comment]
Navigation menu link is pointing to `/en/segwit-upgrade-guide` but permalink is `/en/2016/10/27/segwit-upgrade-guide/`.

**Comment by btcdrak** [comment]
@kanzure I dont understand why, the links are updated in the navigation.yml

**Comment by harding** [comment]
@btcdrak There's a build error:

```
two1-dev:0:~/bitcoincore.org$ bundle exec jekyll serve --future
Configuration file: /home/user/bitcoincore.org/_config.yml
            Source: /home/user/bitcoincore.org
       Destination: /home/user/bitcoincore.org/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
jekyll 3.0.2 | Error:  (/home/user/bitcoincore.org/_data/navigation.yml): did not find expected key while parsing a block mapping at line 128 column 6
```

**Comment by btcdrak** [comment]
@harding thanks, indentation was wrong. Fixed.

**Pull request merged**